cmd/cluster/create/runner.go

```diff
@@ -67,9 +71,72 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		}
 	}
 
+	var configData string
+
+	{
+		// Define nodes.
+		nodes := []KindNode{
+			{
+				Type: "control-plane",
+			},
+		}
+
+		for i := 0; i < r.flag.WorkerCount; i++ {
+			node := KindNode{
+				Type: "worker",
+			}
+
+			nodes = append(nodes, node)
+		}
+
+		data := KindConfig{
+			Nodes: nodes,
+		}
+
+		// Render kind config file.
+		t, err := template.New("kind").Parse(kindConfigTemplate)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		b := new(bytes.Buffer)
+		err = t.Execute(b, data)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		configData = b.String()
+	}
+
+	var configFile afero.File
+
+	defer func() {
```

Comment:
Doesn't it have to show up after we write something on the disk?

Reply:
No, it's just for the cluster create command. After that we don't need it.